docs/linting/README.md

```diff
@@ -85,27 +85,27 @@ With that configured, open a terminal to the root of your project, and run the f
 <TabItem value="npm">
 
 ```bash
-npx eslint . --ext .js,.jsx,.ts,.tsx
+npx eslint . --ext .js,.mjs,.cjs,.jsx,.ts,.mts,.cts,.tsx
```

Comment:
😭 projects are going to copy&paste this and their creators are going to grumble about how annoying all these extensions are. Can we add a notice about only including the extensions needed, maybe?

Reply:
Technically on the latest versions of eslint they won't need this because the overrides config for `eslint-recommended` triggers eslint to automatically include those extensions.

So we could probably remove it and instead link to the CLI docs in case they need custom extensions.

Reply:
Oh 👌 yes, that's excellent.